sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -210,41 +219,67 @@ export class ManagementClient extends LinkEntity {
    */
```

Comment:
TSDoc comments for this method also out of date (before you made this PR in fact!)

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/9520/commits/d8620e01c04132efe5462d71eccee607f0e856cd